dss_datamover/master_application.py

```diff
@@ -84,7 +84,13 @@ def __init__(self, operation, config):
         self.client_password = config["client"]["password"]
 
         self.s3_config = config.get("s3_storage", {})
+
         self.fs_config = self.config.get("fs_config", {})
+        # override NFS configs with CLI args if applicable
+        if self.config['nfs_server'] and self.config['nfs_share']:
```

Comment:
this fails if the user starts app without --nfs-server/--nfs-share options

Reply:
as mentioned internally, it does not, as these options are supplied as keys in self.config with None as values if no CLI options are passed in